### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by chineduG**
Corrected some misspelled words.

 a

### CODE REVIEW

**Comment by Trott** [comment]
Some legitimate fixes mixed in with a lot of style changes and a few wrong fixes. Looks like maybe you're using an automated tool? Would be much easier to handle if this were one type of fix at a time. Like, spelling corrections only, or something like that, one per PR.

File: CHANGELOG.md [hunk 1]
```diff
@@ -219,14 +219,14 @@ introducing the preprocessor directive `NODE_API_SWALLOW_UNTHROWABLE_EXCEPTIONS`
...
-- Added docuemtnation for `Napi::Object::PropertyLValue`.
+- Added documentation for `Napi::Object::PropertyLValue`.
 - Changed all N-API references to Node-API.
 - Some minor corrections all over the documentation.
 
 #### TEST
 
-- Added tests relating to fetch property from Global Object.
-- Added addtiona tests for `Napi::Object`.
+- Added tests relating to fetching property from Global Object.
```
Comment (Trott) [inline CHANGELOG.md:228]: I think this is about a `fetch` property and not about fetching properties, but I'm not sure. @nodejs/node-api 

```suggestion
- Added tests relating to `fetch` property from Global Object.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -321,9 +321,9 @@ conditions of `Napi::AsyncProgressWorker`.
 
 ### TOOL
 
-- Setup github actions for tests.
+- Setup Github actions for tests.
```
Comment (Trott) [inline CHANGELOG.md:324]:
```suggestion
- Setup GitHub Actions for tests.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -321,9 +321,9 @@ conditions of `Napi::AsyncProgressWorker`.
...
 - Added stale action.
-- Removed `sudo` tag from Travis CI.
+- Removed `Sudo` tag from Travis CI.
```
Comment (Trott) [inline CHANGELOG.md:326]:
```suggestion
- Removed `sudo` tag from Travis CI.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -373,10 +373,10 @@ conditions of `Napi::AsyncProgressWorker`.
 #### API
 
 - Introduced `include_dir` for use with **gyp** in a scalar context.
-- Added `Napi::Addon` to help handle the loading of a native add-on into 
+- Added `Napi::Addon` to help handle the loading of a native add-on into
 multiple threads and or multiple times in the same thread.
 - Concentrate callbacks provided to core N-API.
-- Make sure wrapcallback is used.
+- Make sure wrap callback is used.
```
Comment (Trott) [inline CHANGELOG.md:379]: I think this refers to a `wrapcallback` property or somesuch but I'm not sure. @nodejs/node-api 

```suggestion
- Make sure `wrapcallback` is used.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -388,9 +388,9 @@ multiple threads and or multiple times in the same thread.
 #### TEST
 
 - Added tests to check the build process.
-- Refactored test for threasfafe function using async/await.
-- Converted tests that gc into async functions that await 10 ticks after
-each gc.
+- Refactored test for threadsafe function using async/await.
+- Converted tests that go into async functions that await 10 ticks after
+each go.
```
Comment (Trott) [inline CHANGELOG.md:393]:
```suggestion
- Converted tests that `gc` into async functions that await 10 ticks after
each `gc`.
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -636,7 +636,7 @@ are override-able.
 
 #### TEST
 
-- Added test case for bool operator.
+- An added test case for bool operator.
```
Comment (Trott) [inline CHANGELOG.md:639]:
```suggestion
- Added test case for bool operator.
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -667,9 +667,9 @@ are override-able.
 #### API
 
 - Added `SuppressDestruct` method to `Napi::AsyncWorker`.
-- Added new build targets for debug.
+- Added new build targets for debugging.
```
Comment (Trott) [inline CHANGELOG.md:670]: Maybe this?

```suggestion
- Added new build targets for debug builds.
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -667,9 +667,9 @@ are override-able.
...
 - Exposed macros that throw errors.
-- Fixed memory leaks caused by callback data when a napi error occurs.
+- Fixed memory leaks caused by callback data when an api error occurs.
```
Comment (Trott) [inline CHANGELOG.md:672]:
```suggestion
- Fixed memory leaks caused by callback data when a Node-API error occurs.
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -768,7 +768,7 @@ associated with a callback in place when making certain N-API calls
 
 #### Documentation
 
-- Completed the documentation to cover all the API surface.
+- Completed the documentation to cover all the API surfaces.
```
Comment (Trott) [inline CHANGELOG.md:771]:
```suggestion
- Completed the documentation to cover all of the API surface.
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -888,7 +888,7 @@ yet backported in the previous Node.js version.
   - Fixed initialization of std::string to nullptr
 
 #### Tests
-- Fix test failures on linuxOne and AIX
+- Fix test failures on Linux one and AIX
```
Comment (Trott) [inline CHANGELOG.md:891]:
```suggestion
- Fix test failures on LinuxOne and AIX
```

File: CHANGELOG.md [hunk 11]
```diff
@@ -914,3 +914,4 @@ yet backported in the previous Node.js version.
 * [4058a29989] - n-api: fix memory leak in napi_async_destroy() (Jinho Bang) https://github.com/nodejs/node-addon-api/pull/213
 
 
+
```
Comment (Trott) [inline CHANGELOG.md:917]:
```suggestion
````

**Comment by mhdawson** [comment]
Since there has not been a response from the originator for a few months I'm going to close this. Please let us know if that was not the right thing to do.

**Pull request closed without merging**